### PR TITLE
add writePoints to API to match node-influx

### PR DESCRIPTION
**Pull request by anfedorov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,6 @@
     "license": "Apache-2.0",
     "main": "./lib",
     "name": "influx-udp",
-    "repository": "git://github.com/mediocre/node-influx-udp.git",
```
Comment (pwnall) [inline package.json:10]: I think that these changes should be in a separate PR, or should not exist at all. The metadata in `package.json` should only be changed by the releaser... unless you want to change the package name as well?

Given that we're doing a fork, I don't think we need to change the URL or version number. `npm install`-ing with a `git+https` URL in our repo should be sufficient.

Comment (anfedorov) [inline package.json:10]: Good call, I didn't think this through. Removed these changes.

**Comment by pwnall** [comment]
I can't comment on the change to the line protocol. I hope @dorianj can review or assign someone else. Here are some high-level comments:

1) Adding writePoints should be its own PR. I'm saying this based on my observation that the diff doesn't overlap with the line protocol change. If I'm wrong, my claim is no longer true. Please amend the commit to remove the `package.json` changes.
2) Changing protocols should be a separate PR. Also, I think that the explanation in the commit message is a bit terse. It leaves me wondering whether the JSON protocol got deprecated and removed, or whether we disabled it in our server config. If the latter is true, I'd position the PR as switching to the line protocol because it reduces CPU usage on the server (or whatever reason we have for disabling JSON).
3) I think we should PR against upstream. We can put together a feature branch that cherry-picks the PRs, so we don't have to wait for upstream to act.

File: package.json (hunk 1 above)
Comment (pwnall) [inline package.json:10]: Don't be harsh on yourself, semi-open-source is tough!

**Comment by anfedorov** [comment]
sgtm
